pymc_marketing/mmm/mmm.py

```diff
@@ -113,6 +114,17 @@ def __init__(
         adstock_first: bool = Field(
             True, description="Whether to apply adstock first."
         ),
+        dag: str | None = Field(
+            None,
+            description="Optional DAG provided as a string Dot format for causal identification.",
+        ),
+        treatment_nodes: list[str] | tuple[str] | None = Field(
+            None,
+            description="Column names of the variables of interest to identify causal effects on outcome.",
+        ),
+        outcome_node: str | None = Field(
+            None, description="Name of the outcome variable."
+        ),
```

Comment:
Do we have to specify this? We already have the `output_var`. Could that be leveraged?

Reply:
Let me check, not necessarily.

Reply:
If we could simplify, that'd be great

Reply:
@wd60622 we can but then I'll need to restrict that output variable in the DAG is always `y` (The return from `output_var `). Use `sales` or `revenue` or `registrations` or any other would be possible. Not crazy, but then we avoid one parameter. What do you think?